docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: model
+title: BERT Sequence Classification - Identify Antisemitic texts
+author: John Snow Labs
+name: bert_sequence_classifier_antisemitism
+date: 2021-11-06
+tags: [en, open_source]
+task: Text Classification
+language: en
+edition: Spark NLP 3.3.2
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is imported from `Hugging Face-models and it was trained on 4K tweets, where ~50% were labeled as antisemitic. The model identifies if the text is antisemitic or not. 
```

Comment:
Missing a close quote after Hugging Face-models

Reply:
yes it should be like this -->  `Hugging Face-models`